Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -321,6 +321,11 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"INVALID_MASTERSERVER_TOKEN" 	"Invalid or expired masterserver token"
 		"JSON_PARSE_ERROR" 				"Error parsing json response"
 		"UNSUPPORTED_VERSION" 			"The version you are using is no longer supported"
+
+		"AUTHENTICATION_FAILED_HEADER"		"Authentication Failed"
+		"AUTHENTICATION_FAILED_BODY"		"Failed to authenticate with Atlas!"
```

Comment:
no one who cant fix it themself knows what atlas is, I feel like Masterserver is more understanable

Reply:
maaaaaan, we really need to start moving away from calling it the masterserver when it is called Atlas now

Reply:
its not for me okay, its for the northstar chat users, lets hear a 3rd party on this